src/util/cmdline.h

```diff
@@ -20,6 +20,58 @@ Author: Daniel Kroening, kroening@kroening.com
 class cmdlinet
 {
 public:
+  /// Parses a commandline according to a specification given in optstring.
+  /// \param argc How many arguments there are.
+  /// \param argv An array of C strings.
+  ///             The 0th element is assumed to be the name of the command as
+  ///             it was invoked (e.g. /usr/bin/cmake) and is ignored. It is
+  ///             further assumed the array holds argc+1 elements with the C
+  ///             string at index argc being a terminating null pointer.
+  ///             This parameter is parsed based on optstring.
+  /// \param optstring A specification of allowed command line options.
+  ///                  This is a C string container any number of single
+  ///                  characters other than '(', ')' or ':' signifying a
+  ///                  "short" option consisting of just that character, or
+  ///                  names consisting of any characters other than ')'
+  ///                  surrounded by a matching pair of '(' and ')' signifying a
+  ///                  "long" option with the name being the string between '('
+  ///                  and ')', both of which can be optionally followed by a
+  ///                  single ':' indicating that the option takes a parameter,
```

Comment:
In my opinion, all further uses of "parameter" should be replaced by "argument" here.